clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -45,10 +58,26 @@ public Options options() {
     options.addOption(createSimpleOption("v", VERSION, "Gravitino client version"));
     options.addOption(createSimpleOption("r", SERVER, "Gravitino server version"));
     options.addOption(createArgOption("u", URL, "Gravitino URL (default: http://localhost:8090)"));
-    options.addOption(createArgOption("f", NAME, "full entity name (dot separated)"));
+    options.addOption(createArgOption("n", NAME, "full entity name (dot separated)"));
     options.addOption(createArgOption("m", METALAKE, "Metalake name"));
     options.addOption(createSimpleOption("i", IGNORE, "Ignore client/sever version check"));
 
+    // Create/update options
+    options.addOption(createArgOption("r", RENAME, "new entity name"));
+    options.addOption(createArgOption("c", COMMENT, "entity comment"));
+    options.addOption(createArgOption("p", PROPERTY, "property name"));
+    options.addOption(createArgOption("v", VALUE, "property value"));
+    options.addOption(
+        createArgOption(
+            "p", PROVIDER, "provider one of hadoop, hive, mysql, postgres, iceberg, kafka"));
```

Comment:
The short-name of "p" has been occupied by "property" above?

Reply:
Which is fine as a) short options are generally not used b) they are not exclusive and c) both options can't be used together.

Reply:
I think it is confusing that a name has ambiguity. "short options are generally not used" this is not a good reason, otherwise why we support it. Name conflict is strange, the user don't know what's the behavior; At lease we can use lower-case and upper-case to differenciate them. I'm not expert on this, but that's my concern, unless you can point to me that other CLI (like linux command) also has such duplicated names... 

BTW, please don't mark it resolved quickly before the reviewer see the comment it. Thanks.

Reply:
Given the other discussion about removing the named catalog creation options, this is no longer an issue.

Reply:
I've reviewed and changed the options that were left so their are no duplicates.

Reply:
I see, now there is no duplication.